src/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+// import React from 'react';
```

Comment:
Yup...I think this may be unnecessary in the current version of React. We can leave or remove.